x-pack/legacy/plugins/uptime/public/components/settings/certificate_form.tsx

```diff
@@ -55,15 +53,15 @@ export const CertificateExpirationForm: React.FC<SettingsFormProps> = ({
         title={
           <h4>
             <FormattedMessage
-              id="xpack.uptime.sourceConfiguration.stateThresholds"
-              defaultMessage="Expiration State Thresholds"
+              id="xpack.uptime.sourceConfiguration.expirationThreshold"
+              defaultMessage="Expiration/Age Thresholds"
```

Comment:
we should also add that text here which got updated in ACs "Change the threshold for displaying and alerting on certificate errors. Note, this will affect any configured alerts"

Reply:
I added this copy in https://github.com/elastic/kibana/pull/64111/commits/0ddd97b6031717ab5176f3fa5c9d8cfa4e435529, modified the punctuation a bit in https://github.com/elastic/kibana/pull/64111/commits/a1fdf46997721a18cc7b5308313da8096e1a8147.